diff -ru cd-hit-v4.6.1-2012-08-27-orig/cd-hit-2d-para.pl cd-hit-v4.6.1-2012-08-27/cd-hit-2d-para.pl
--- cd-hit-v4.6.1-2012-08-27-orig/cd-hit-2d-para.pl	2012-10-02 18:29:45.291241094 -0400
+++ cd-hit-v4.6.1-2012-08-27/cd-hit-2d-para.pl	2012-10-02 18:32:49.344687732 -0400
@@ -1,4 +1,5 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
+use warnings;
 # =============================================================================
 # CD-HIT
 # http://cd-hit.org/
diff -ru cd-hit-v4.6.1-2012-08-27-orig/cd-hit-div.pl cd-hit-v4.6.1-2012-08-27/cd-hit-div.pl
--- cd-hit-v4.6.1-2012-08-27-orig/cd-hit-div.pl	2012-10-02 18:29:45.347831813 -0400
+++ cd-hit-v4.6.1-2012-08-27/cd-hit-div.pl	2012-10-02 18:32:49.351943387 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 #not like cd-hit-div, this script do not sort input
 #or throw away seq
diff -ru cd-hit-v4.6.1-2012-08-27-orig/cd-hit-para.pl cd-hit-v4.6.1-2012-08-27/cd-hit-para.pl
--- cd-hit-v4.6.1-2012-08-27-orig/cd-hit-para.pl	2012-10-02 18:29:45.309522299 -0400
+++ cd-hit-v4.6.1-2012-08-27/cd-hit-para.pl	2012-10-02 18:32:49.359536314 -0400
@@ -1,4 +1,5 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
+use warnings;
 # =============================================================================
 # CD-HIT
 # http://cd-hit.org/
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr2tree.pl cd-hit-v4.6.1-2012-08-27/clstr2tree.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr2tree.pl	2012-10-02 18:29:45.178261880 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr2tree.pl	2012-10-02 18:32:49.366191655 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 $clstr = shift;
 $fr = shift; # for nr80.clstr $fr = 0.8
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr2txt.pl cd-hit-v4.6.1-2012-08-27/clstr2txt.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr2txt.pl	2012-10-02 18:29:45.316669650 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr2txt.pl	2012-10-02 18:32:49.373037208 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $no = 0;
 my $clstr_no = "";
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr2xml.pl cd-hit-v4.6.1-2012-08-27/clstr2xml.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr2xml.pl	2012-10-02 18:29:45.174561326 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr2xml.pl	2012-10-02 18:32:49.379831089 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 #usage: clstr_xml.pl [-len|-size] level1.clstr [level2.clstr level3.clstr ...]
 #purpose: to create xml file from cd-hit or hierarchical cd-hit(h-cd-hit) results
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_cut.pl cd-hit-v4.6.1-2012-08-27/clstr_cut.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_cut.pl	2012-10-02 18:29:45.197122730 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_cut.pl	2012-10-02 18:32:49.386275478 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 #keep only top $no proteins in cluster
 
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_merge_noorder.pl cd-hit-v4.6.1-2012-08-27/clstr_merge_noorder.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_merge_noorder.pl	2012-10-02 18:29:45.313069161 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_merge_noorder.pl	2012-10-02 18:32:49.392838436 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 # order of clusters don't need to be the same
 # but then I have to read everything into memory
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_merge.pl cd-hit-v4.6.1-2012-08-27/clstr_merge.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_merge.pl	2012-10-02 18:29:45.264510659 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_merge.pl	2012-10-02 18:32:49.400298340 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 # the order of clusters need to be identical
 my ($master_clstr, @clstr) = @ARGV;
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_quality_eval_by_link.pl cd-hit-v4.6.1-2012-08-27/clstr_quality_eval_by_link.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_quality_eval_by_link.pl	2012-10-02 18:29:45.104564610 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_quality_eval_by_link.pl	2012-10-02 18:32:49.406825127 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 ## calculate the sensitivity and specificity of clusters
 ## if the input fasta file has pre-defined classification term
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_quality_eval.pl cd-hit-v4.6.1-2012-08-27/clstr_quality_eval.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_quality_eval.pl	2012-10-02 18:29:45.275743667 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_quality_eval.pl	2012-10-02 18:32:49.414583826 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 ## calculate the sensitivity and specificity of clusters
 ## if the input fasta file has pre-defined classification term
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_reduce.pl cd-hit-v4.6.1-2012-08-27/clstr_reduce.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_reduce.pl	2012-10-02 18:29:45.340382314 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_reduce.pl	2012-10-02 18:32:49.421477421 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 
 $file90 = shift;
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_renumber.pl cd-hit-v4.6.1-2012-08-27/clstr_renumber.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_renumber.pl	2012-10-02 18:29:45.185624093 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_renumber.pl	2012-10-02 18:32:49.427907915 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 $no = 0;
 while($ll=<>){
   if ($ll =~ /^>Cluster (\d+)/) {
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_rep.pl cd-hit-v4.6.1-2012-08-27/clstr_rep.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_rep.pl	2012-10-02 18:29:45.356106404 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_rep.pl	2012-10-02 18:32:49.434153154 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 $rep = "";
 $no = 0;
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_reps_faa_rev.pl cd-hit-v4.6.1-2012-08-27/clstr_reps_faa_rev.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_reps_faa_rev.pl	2012-10-02 18:29:45.188909378 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_reps_faa_rev.pl	2012-10-02 18:32:49.440478486 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 # output single fasta file
 # for each cluster output at least $cutoff seqs
 
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_rev.pl cd-hit-v4.6.1-2012-08-27/clstr_rev.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_rev.pl	2012-10-02 18:29:45.101003712 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_rev.pl	2012-10-02 18:32:49.447055619 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 # if nr90 from nr100 and
 #    nr80 from nr90, so I have nr90.clstr and nr80.clstr
 # but, in nr80.clstr, some gi numbers whose from nr100 are there
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_select.pl cd-hit-v4.6.1-2012-08-27/clstr_select.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_select.pl	2012-10-02 18:29:45.128942738 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_select.pl	2012-10-02 18:32:49.453677582 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 #my $by = shift;
 my $min;
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_select_rep.pl cd-hit-v4.6.1-2012-08-27/clstr_select_rep.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_select_rep.pl	2012-10-02 18:29:45.305783618 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_select_rep.pl	2012-10-02 18:32:49.460506167 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 #my $by = shift;
 my $min;
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_size_histogram.pl cd-hit-v4.6.1-2012-08-27/clstr_size_histogram.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_size_histogram.pl	2012-10-02 18:29:45.294788236 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_size_histogram.pl	2012-10-02 18:32:49.467505343 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 if(@ARGV==0){
    print "Usage:\n\tclstr_size_histogram.pl [-bin N] clstr_file\n";
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_size_stat.pl cd-hit-v4.6.1-2012-08-27/clstr_size_stat.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_size_stat.pl	2012-10-02 18:29:45.160722323 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_size_stat.pl	2012-10-02 18:32:49.474400894 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 if(@ARGV==0){
    print "Usage:\n\tclstr_size_stat.pl clstr_file\n";
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_sort_by.pl cd-hit-v4.6.1-2012-08-27/clstr_sort_by.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_sort_by.pl	2012-10-02 18:29:45.351992117 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_sort_by.pl	2012-10-02 18:32:49.481025022 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $sort_by_what = shift;
    $sort_by_what = "no" unless $sort_by_what;
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_sort_prot_by.pl cd-hit-v4.6.1-2012-08-27/clstr_sort_prot_by.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_sort_prot_by.pl	2012-10-02 18:29:45.211698282 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_sort_prot_by.pl	2012-10-02 18:32:49.487688184 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $sort_by = shift;
    $sort_by = "len" unless ($sort_by);
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_sql_tbl.pl cd-hit-v4.6.1-2012-08-27/clstr_sql_tbl.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_sql_tbl.pl	2012-10-02 18:29:45.108132211 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_sql_tbl.pl	2012-10-02 18:32:49.494659568 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 if(@ARGV==0){
   print "Usage:\n\tclstr_sql_tbl.pl clstr_file tbl_file\n";
diff -ru cd-hit-v4.6.1-2012-08-27-orig/clstr_sql_tbl_sort.pl cd-hit-v4.6.1-2012-08-27/clstr_sql_tbl_sort.pl
--- cd-hit-v4.6.1-2012-08-27-orig/clstr_sql_tbl_sort.pl	2012-10-02 18:29:45.156880506 -0400
+++ cd-hit-v4.6.1-2012-08-27/clstr_sql_tbl_sort.pl	2012-10-02 18:32:49.501201228 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 if(@ARGV==0){
    print "Usage:\n\tclstr_sql_tbl_sort.pl table_file level\n";
diff -ru cd-hit-v4.6.1-2012-08-27-orig/make_multi_seq.pl cd-hit-v4.6.1-2012-08-27/make_multi_seq.pl
--- cd-hit-v4.6.1-2012-08-27-orig/make_multi_seq.pl	2012-10-02 18:29:45.111835837 -0400
+++ cd-hit-v4.6.1-2012-08-27/make_multi_seq.pl	2012-10-02 18:32:49.507579769 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 #note you have to use "-d 0" in the cd-hit run
 #note you better to use "-g 1" in the cd-hit run
diff -ru cd-hit-v4.6.1-2012-08-27-orig/plot_2d.pl cd-hit-v4.6.1-2012-08-27/plot_2d.pl
--- cd-hit-v4.6.1-2012-08-27-orig/plot_2d.pl	2012-10-02 18:29:45.201239810 -0400
+++ cd-hit-v4.6.1-2012-08-27/plot_2d.pl	2012-10-02 18:32:49.514280499 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 use Image::Magick;
 
diff -ru cd-hit-v4.6.1-2012-08-27-orig/plot_len1.pl cd-hit-v4.6.1-2012-08-27/plot_len1.pl
--- cd-hit-v4.6.1-2012-08-27-orig/plot_len1.pl	2012-10-02 18:29:45.359868128 -0400
+++ cd-hit-v4.6.1-2012-08-27/plot_len1.pl	2012-10-02 18:32:49.520994635 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 $file90 = shift;
 $segs = shift;
diff -ru cd-hit-v4.6.1-2012-08-27-orig/psi-cd-hit-2d-g1.pl cd-hit-v4.6.1-2012-08-27/psi-cd-hit-2d-g1.pl
--- cd-hit-v4.6.1-2012-08-27-orig/psi-cd-hit-2d-g1.pl	2012-10-02 18:29:45.324067686 -0400
+++ cd-hit-v4.6.1-2012-08-27/psi-cd-hit-2d-g1.pl	2012-10-02 18:32:49.527556127 -0400
@@ -1,4 +1,5 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
+use warnings;
 no warnings qw(once);
 
 our $script_name = $0;
diff -ru cd-hit-v4.6.1-2012-08-27-orig/psi-cd-hit-2d.pl cd-hit-v4.6.1-2012-08-27/psi-cd-hit-2d.pl
--- cd-hit-v4.6.1-2012-08-27-orig/psi-cd-hit-2d.pl	2012-10-02 18:29:45.287509048 -0400
+++ cd-hit-v4.6.1-2012-08-27/psi-cd-hit-2d.pl	2012-10-02 18:32:49.534238143 -0400
@@ -1,4 +1,5 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
+use warnings;
 
 package PSI_CDHIT;
 
diff -ru cd-hit-v4.6.1-2012-08-27-orig/psi-cd-hit-local.pl cd-hit-v4.6.1-2012-08-27/psi-cd-hit-local.pl
--- cd-hit-v4.6.1-2012-08-27-orig/psi-cd-hit-local.pl	2012-10-02 18:29:45.165477215 -0400
+++ cd-hit-v4.6.1-2012-08-27/psi-cd-hit-local.pl	2012-10-02 18:32:49.543432373 -0400
@@ -1,4 +1,5 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
+use warnings;
 
 no warnings qw(once);
 
@@ -1956,7 +1957,7 @@
 
   open(REPERL, "> $remote_perl_script") || die;
   print REPERL <<EOD;
-#!/usr/bin/perl
+#!/usr/bin/env perl
 \$host = shift;
 \$arg = shift;
 
diff -ru cd-hit-v4.6.1-2012-08-27-orig/psi-cd-hit.pl cd-hit-v4.6.1-2012-08-27/psi-cd-hit.pl
--- cd-hit-v4.6.1-2012-08-27-orig/psi-cd-hit.pl	2012-10-02 18:29:45.298524611 -0400
+++ cd-hit-v4.6.1-2012-08-27/psi-cd-hit.pl	2012-10-02 18:32:49.550410810 -0400
@@ -1,4 +1,5 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
+use warnings;
 
 package PSI_CDHIT;
 
