diff -ru cd-hit-otu-454-0.0.2-orig/cd-hit-otu-all.pl cd-hit-otu-454-0.0.2/cd-hit-otu-all.pl
--- cd-hit-otu-454-0.0.2-orig/cd-hit-otu-all.pl	2012-10-02 19:14:47.713000424 -0400
+++ cd-hit-otu-454-0.0.2/cd-hit-otu-all.pl	2012-10-02 19:15:05.345032169 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 use Getopt::Std;
 my $script_name = $0;
diff -ru cd-hit-otu-454-0.0.2-orig/cd-hit-otu-all.v0.pl cd-hit-otu-454-0.0.2/cd-hit-otu-all.v0.pl
--- cd-hit-otu-454-0.0.2-orig/cd-hit-otu-all.v0.pl	2012-10-02 19:14:47.803598881 -0400
+++ cd-hit-otu-454-0.0.2/cd-hit-otu-all.v0.pl	2012-10-02 19:15:05.351885474 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 use Getopt::Std;
 my $script_name = $0;
diff -ru cd-hit-otu-454-0.0.2-orig/cd-hit-otu-simulation.pl cd-hit-otu-454-0.0.2/cd-hit-otu-simulation.pl
--- cd-hit-otu-454-0.0.2-orig/cd-hit-otu-simulation.pl	2012-10-02 19:14:47.701925646 -0400
+++ cd-hit-otu-454-0.0.2/cd-hit-otu-simulation.pl	2012-10-02 19:15:05.358869916 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $len= $ARGV[0];
 my $err= $ARGV[1];
diff -ru cd-hit-otu-454-0.0.2-orig/cd-hit-otu-simulation-pyro.pl cd-hit-otu-454-0.0.2/cd-hit-otu-simulation-pyro.pl
--- cd-hit-otu-454-0.0.2-orig/cd-hit-otu-simulation-pyro.pl	2012-10-02 19:14:47.732655850 -0400
+++ cd-hit-otu-454-0.0.2/cd-hit-otu-simulation-pyro.pl	2012-10-02 19:15:05.365917832 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $err     = shift;
 my $len     = shift;
diff -ru cd-hit-otu-454-0.0.2-orig/cd-hit-otu-step1-filter-by-primers.pl cd-hit-otu-454-0.0.2/cd-hit-otu-step1-filter-by-primers.pl
--- cd-hit-otu-454-0.0.2-orig/cd-hit-otu-step1-filter-by-primers.pl	2012-10-02 19:14:47.963483830 -0400
+++ cd-hit-otu-454-0.0.2/cd-hit-otu-step1-filter-by-primers.pl	2012-10-02 19:15:05.372445526 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $fasta = shift;
 my $primer_file = shift;
diff -ru cd-hit-otu-454-0.0.2-orig/cd-hit-otu-step1-filter-trim.pl cd-hit-otu-454-0.0.2/cd-hit-otu-step1-filter-trim.pl
--- cd-hit-otu-454-0.0.2-orig/cd-hit-otu-step1-filter-trim.pl	2012-10-02 19:14:47.952786823 -0400
+++ cd-hit-otu-454-0.0.2/cd-hit-otu-step1-filter-trim.pl	2012-10-02 19:15:05.379227536 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 use Getopt::Std;
 
diff -ru cd-hit-otu-454-0.0.2-orig/clstr_sample_count_matrix.pl cd-hit-otu-454-0.0.2/clstr_sample_count_matrix.pl
--- cd-hit-otu-454-0.0.2-orig/clstr_sample_count_matrix.pl	2012-10-02 19:14:47.959993669 -0400
+++ cd-hit-otu-454-0.0.2/clstr_sample_count_matrix.pl	2012-10-02 19:15:05.385865839 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $delimiter = shift;
 my $clstr = shift;
diff -ru cd-hit-otu-454-0.0.2-orig/clstr_sample_count.pl cd-hit-otu-454-0.0.2/clstr_sample_count.pl
--- cd-hit-otu-454-0.0.2-orig/clstr_sample_count.pl	2012-10-02 19:14:47.795602067 -0400
+++ cd-hit-otu-454-0.0.2/clstr_sample_count.pl	2012-10-02 19:15:05.392467412 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $delimiter = shift;
 my $clstr = shift;
diff -ru cd-hit-otu-454-0.0.2-orig/clstr_select_pri.pl cd-hit-otu-454-0.0.2/clstr_select_pri.pl
--- cd-hit-otu-454-0.0.2-orig/clstr_select_pri.pl	2012-10-02 19:14:47.949106240 -0400
+++ cd-hit-otu-454-0.0.2/clstr_select_pri.pl	2012-10-02 19:15:05.399224912 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $clstr = shift;
 my $fasta = shift;
diff -ru cd-hit-otu-454-0.0.2-orig/clstr_select_rep.pl cd-hit-otu-454-0.0.2/clstr_select_rep.pl
--- cd-hit-otu-454-0.0.2-orig/clstr_select_rep.pl	2012-10-02 19:14:47.814263976 -0400
+++ cd-hit-otu-454-0.0.2/clstr_select_rep.pl	2012-10-02 19:15:05.405609459 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $by = shift;
 my $min;
diff -ru cd-hit-otu-454-0.0.2-orig/clstr_sort_trim_rep.pl cd-hit-otu-454-0.0.2/clstr_sort_trim_rep.pl
--- cd-hit-otu-454-0.0.2-orig/clstr_sort_trim_rep.pl	2012-10-02 19:14:47.705569709 -0400
+++ cd-hit-otu-454-0.0.2/clstr_sort_trim_rep.pl	2012-10-02 19:15:05.412085760 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $clstr = shift;
 my $fasta = shift;
diff -ru cd-hit-otu-454-0.0.2-orig/clstr_vs_clstr_plot.pl cd-hit-otu-454-0.0.2/clstr_vs_clstr_plot.pl
--- cd-hit-otu-454-0.0.2-orig/clstr_vs_clstr_plot.pl	2012-10-02 19:14:47.967045146 -0400
+++ cd-hit-otu-454-0.0.2/clstr_vs_clstr_plot.pl	2012-10-02 19:15:05.418820985 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $clstr_ref=shift;
 my $clstr_sbj=shift;
diff -ru cd-hit-otu-454-0.0.2-orig/eval2.pl cd-hit-otu-454-0.0.2/eval2.pl
--- cd-hit-otu-454-0.0.2-orig/eval2.pl	2012-10-02 19:14:47.709297636 -0400
+++ cd-hit-otu-454-0.0.2/eval2.pl	2012-10-02 19:15:05.425378775 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $ref = shift; #fasta of reference 
 my $raw = shift; #raw reads
diff -ru cd-hit-otu-454-0.0.2-orig/evala.pl cd-hit-otu-454-0.0.2/evala.pl
--- cd-hit-otu-454-0.0.2-orig/evala.pl	2012-10-02 19:14:47.956451485 -0400
+++ cd-hit-otu-454-0.0.2/evala.pl	2012-10-02 19:15:05.431932656 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $ref = shift; #fasta of reference 
 my $raw = shift; #raw reads
diff -ru cd-hit-otu-454-0.0.2-orig/eval.pl cd-hit-otu-454-0.0.2/eval.pl
--- cd-hit-otu-454-0.0.2-orig/eval.pl	2012-10-02 19:14:47.817996371 -0400
+++ cd-hit-otu-454-0.0.2/eval.pl	2012-10-02 19:15:05.438383888 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $ref = shift; #fasta of reference 
 my $raw = shift; #raw reads
diff -ru cd-hit-otu-454-0.0.2-orig/fasta_reverse_strand.pl cd-hit-otu-454-0.0.2/fasta_reverse_strand.pl
--- cd-hit-otu-454-0.0.2-orig/fasta_reverse_strand.pl	2012-10-02 19:14:47.810735968 -0400
+++ cd-hit-otu-454-0.0.2/fasta_reverse_strand.pl	2012-10-02 19:15:05.444920661 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $fasta = shift;
 my $output = "$fasta.rev";
diff -ru cd-hit-otu-454-0.0.2-orig/fetch_fasta_by_ids.pl cd-hit-otu-454-0.0.2/fetch_fasta_by_ids.pl
--- cd-hit-otu-454-0.0.2-orig/fetch_fasta_by_ids.pl	2012-10-02 19:14:47.807130869 -0400
+++ cd-hit-otu-454-0.0.2/fetch_fasta_by_ids.pl	2012-10-02 19:15:05.451438021 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my ($gi_file, $seq_file) = @ARGV;
 
diff -ru cd-hit-otu-454-0.0.2-orig/fetch_fasta_exclude_ids.pl cd-hit-otu-454-0.0.2/fetch_fasta_exclude_ids.pl
--- cd-hit-otu-454-0.0.2-orig/fetch_fasta_exclude_ids.pl	2012-10-02 19:14:47.799373287 -0400
+++ cd-hit-otu-454-0.0.2/fetch_fasta_exclude_ids.pl	2012-10-02 19:15:05.457983662 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my ($gi_file, $seq_file) = @ARGV;
 
diff -ru cd-hit-otu-454-0.0.2-orig/scan_fastq.pl cd-hit-otu-454-0.0.2/scan_fastq.pl
--- cd-hit-otu-454-0.0.2-orig/scan_fastq.pl	2012-10-02 19:14:47.724426648 -0400
+++ cd-hit-otu-454-0.0.2/scan_fastq.pl	2012-10-02 19:15:05.464490129 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $no = 0;
 my $len = 0;
diff -ru cd-hit-otu-454-0.0.2-orig/scan_qual.pl cd-hit-otu-454-0.0.2/scan_qual.pl
--- cd-hit-otu-454-0.0.2-orig/scan_qual.pl	2012-10-02 19:14:47.716504830 -0400
+++ cd-hit-otu-454-0.0.2/scan_qual.pl	2012-10-02 19:15:05.470809595 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 my $total_score = 0;
 my $total_base = 0;
diff -ru cd-hit-otu-454-0.0.2-orig/trim-long-seq.pl cd-hit-otu-454-0.0.2/trim-long-seq.pl
--- cd-hit-otu-454-0.0.2-orig/trim-long-seq.pl	2012-10-02 19:14:47.970525322 -0400
+++ cd-hit-otu-454-0.0.2/trim-long-seq.pl	2012-10-02 19:15:05.477394549 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 use Getopt::Std;
 
diff -ru cd-hit-otu-454-0.0.2-orig/z1.pl cd-hit-otu-454-0.0.2/z1.pl
--- cd-hit-otu-454-0.0.2-orig/z1.pl	2012-10-02 19:14:47.720157691 -0400
+++ cd-hit-otu-454-0.0.2/z1.pl	2012-10-02 19:15:05.484450496 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 use Getopt::Std;
 my $script_name = $0;
