diff -ruN trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/CDNA/Alternative_splice_comparer.pm trinityrnaseq-Trinity-v2.8.4/PerlLib/CDNA/Alternative_splice_comparer.pm
--- trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/CDNA/Alternative_splice_comparer.pm	2018-09-12 06:08:03.000000000 -0400
+++ trinityrnaseq-Trinity-v2.8.4/PerlLib/CDNA/Alternative_splice_comparer.pm	2018-11-08 16:17:07.521174999 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/env perl
 
 package main;
 our $SEE;
diff -ruN trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/CDNA/CDNA_alignment.pm trinityrnaseq-Trinity-v2.8.4/PerlLib/CDNA/CDNA_alignment.pm
--- trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/CDNA/CDNA_alignment.pm	2018-09-12 06:08:03.000000000 -0400
+++ trinityrnaseq-Trinity-v2.8.4/PerlLib/CDNA/CDNA_alignment.pm	2018-11-08 16:17:07.558243284 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/env perl
 
 ##########################
 ### Class CDNA_alignment
diff -ruN trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/CDNA/Genome_based_cDNA_assembler.pm trinityrnaseq-Trinity-v2.8.4/PerlLib/CDNA/Genome_based_cDNA_assembler.pm
--- trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/CDNA/Genome_based_cDNA_assembler.pm	2018-09-12 06:08:03.000000000 -0400
+++ trinityrnaseq-Trinity-v2.8.4/PerlLib/CDNA/Genome_based_cDNA_assembler.pm	2018-11-08 16:17:07.544568172 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/env perl
 
 package main;
 our $SEE;
diff -ruN trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/CDNA/Genome_based_cDNA_graph_assembler.pm trinityrnaseq-Trinity-v2.8.4/PerlLib/CDNA/Genome_based_cDNA_graph_assembler.pm
--- trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/CDNA/Genome_based_cDNA_graph_assembler.pm	2018-09-12 06:08:03.000000000 -0400
+++ trinityrnaseq-Trinity-v2.8.4/PerlLib/CDNA/Genome_based_cDNA_graph_assembler.pm	2018-11-08 16:17:07.591413300 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/env perl
 
 package main;
 our $SEE;
diff -ruN trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/CDNA/PASA_alignment_assembler.pm trinityrnaseq-Trinity-v2.8.4/PerlLib/CDNA/PASA_alignment_assembler.pm
--- trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/CDNA/PASA_alignment_assembler.pm	2018-09-12 06:08:03.000000000 -0400
+++ trinityrnaseq-Trinity-v2.8.4/PerlLib/CDNA/PASA_alignment_assembler.pm	2018-11-08 16:17:07.498925645 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/env perl
 
 package main;
 our $SEE;
diff -ruN trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/CDNA/Splice_graph_assembler.pm trinityrnaseq-Trinity-v2.8.4/PerlLib/CDNA/Splice_graph_assembler.pm
--- trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/CDNA/Splice_graph_assembler.pm	2018-09-12 06:08:03.000000000 -0400
+++ trinityrnaseq-Trinity-v2.8.4/PerlLib/CDNA/Splice_graph_assembler.pm	2018-11-08 16:17:07.477833659 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/env perl
 
 package main;
 our $SEE;
diff -ruN trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/Fasta_reader.pm trinityrnaseq-Trinity-v2.8.4/PerlLib/Fasta_reader.pm
--- trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/Fasta_reader.pm	2018-09-12 06:08:03.000000000 -0400
+++ trinityrnaseq-Trinity-v2.8.4/PerlLib/Fasta_reader.pm	2018-11-08 16:17:07.800558011 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl -w
+#!/usr/bin/env perl
 
 # lightweight fasta reader capabilities:
 package Fasta_reader;
diff -ruN trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/Gene_obj_indexer.pm trinityrnaseq-Trinity-v2.8.4/PerlLib/Gene_obj_indexer.pm
--- trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/Gene_obj_indexer.pm	2018-09-12 06:08:03.000000000 -0400
+++ trinityrnaseq-Trinity-v2.8.4/PerlLib/Gene_obj_indexer.pm	2018-11-08 16:17:08.093002989 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/env perl
 
 package Gene_obj_indexer;
 use strict;
diff -ruN trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/GFF3_utils.pm trinityrnaseq-Trinity-v2.8.4/PerlLib/GFF3_utils.pm
--- trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/GFF3_utils.pm	2018-09-12 06:08:03.000000000 -0400
+++ trinityrnaseq-Trinity-v2.8.4/PerlLib/GFF3_utils.pm	2018-11-08 16:17:08.113731306 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/env perl
 
 package main;
 our $SEE;
diff -ruN trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/GFF_maker.pm trinityrnaseq-Trinity-v2.8.4/PerlLib/GFF_maker.pm
--- trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/GFF_maker.pm	2018-09-12 06:08:03.000000000 -0400
+++ trinityrnaseq-Trinity-v2.8.4/PerlLib/GFF_maker.pm	2018-11-08 16:17:07.441024526 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/env perl
 
 package main;
 our $SEE;
diff -ruN trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/Longest_orf.pm trinityrnaseq-Trinity-v2.8.4/PerlLib/Longest_orf.pm
--- trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/Longest_orf.pm	2018-09-12 06:08:03.000000000 -0400
+++ trinityrnaseq-Trinity-v2.8.4/PerlLib/Longest_orf.pm	2018-11-08 16:17:07.679186341 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/env perl
 
 package main;
 our $SEE; 
diff -ruN trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/overlapping_nucs.ph trinityrnaseq-Trinity-v2.8.4/PerlLib/overlapping_nucs.ph
--- trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/overlapping_nucs.ph	2018-09-12 06:08:03.000000000 -0400
+++ trinityrnaseq-Trinity-v2.8.4/PerlLib/overlapping_nucs.ph	2018-11-08 16:17:08.156316224 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/env perl
 
 ####
 sub nucs_in_common {
diff -ruN trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/TiedHash.pm trinityrnaseq-Trinity-v2.8.4/PerlLib/TiedHash.pm
--- trinityrnaseq-Trinity-v2.8.4-orig/PerlLib/TiedHash.pm	2018-09-12 06:08:03.000000000 -0400
+++ trinityrnaseq-Trinity-v2.8.4/PerlLib/TiedHash.pm	2018-11-08 16:17:07.720846957 -0500
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/env perl
 
 package TiedHash;
 use strict;
