diff -uNr ../SOURCES/garli-0.96b8-orig/src/bipartition.h garli-0.96b8/src/bipartition.h
--- ../SOURCES/garli-0.96b8-orig/src/bipartition.h	2012-08-17 14:02:00.069801232 -0400
+++ garli-0.96b8/src/bipartition.h	2012-08-17 13:59:41.339814730 -0400
@@ -21,6 +21,7 @@
 
 #include <cmath>
 #include <string>
+#include <cstring>
 #include <vector>
 #include <cassert>
 
diff -uNr ../SOURCES/garli-0.96b8-orig/src/configoptions.h garli-0.96b8/src/configoptions.h
--- ../SOURCES/garli-0.96b8-orig/src/configoptions.h	2012-08-17 14:02:00.071821582 -0400
+++ garli-0.96b8/src/configoptions.h	2012-08-17 14:01:22.548835777 -0400
@@ -20,6 +20,7 @@
 #define CONFIGOPTIONS_H
 
 #include <string>
+#include <limits.h>
 
 using std::string;
 
diff -uNr ../SOURCES/garli-0.96b8-orig/src/garlireader.h garli-0.96b8/src/garlireader.h
--- ../SOURCES/garli-0.96b8-orig/src/garlireader.h	2012-08-17 14:02:00.070821964 -0400
+++ garli-0.96b8/src/garlireader.h	2012-08-17 14:04:25.317842971 -0400
@@ -43,6 +43,7 @@
 #define COMMAND_MAXLEN  255
 
 #include "ncl.h"
+#include "assert.h"
 
 /*----------------------------------------------------------------------------------------------------------------------
 |	GarliReader provides a template for creating a program that reads NEXUS data files and provides a basic command 
diff -uNr ../SOURCES/garli-0.96b8-orig/src/translatetable.h garli-0.96b8/src/translatetable.h
--- ../SOURCES/garli-0.96b8-orig/src/translatetable.h	2012-08-17 14:02:00.070821964 -0400
+++ garli-0.96b8/src/translatetable.h	2012-08-17 14:05:12.714845100 -0400
@@ -22,6 +22,7 @@
 
 #include <cassert>
 #include <string>
+#include <cstring>
 #include <iostream>
 
 using namespace std;
