--- breakdancer/perl/BreakDancerMax.pl.orig	2012-10-15 09:51:41.566832328 -0400
+++ breakdancer/perl/BreakDancerMax.pl	2012-10-15 09:55:02.423571569 -0400
@@ -1,11 +1,11 @@
-#!/gsc/bin/perl
+#!/usr/bin/env perl
 #joint analysis of multiple maps (lib specific)
 #  Copyright (C) 2009 Washington University in St. Louis
 
 use strict;
 use warnings;
 use Getopt::Std;
-use lib '/gscuser/kchen/1000genomes/analysis/scripts/';
+use lib '/opt/bio/breakdancer/perl';
 use Poisson;
 use AlnParser;
 use Statistics::Descriptive;
