--- bin/Makefile.am	2015-10-29 14:15:19.924387929 -0400
+++ bin/Makefile.am	2015-10-29 14:15:57.225749038 -0400
@@ -1,6 +1,14 @@
 dist_bin_SCRIPTS = \
 	sga-astat.py \
 	sga-bam2de.pl \
-    sga-mergeDriver.pl 
-dist_noinst_SCRIPTS=sga-align sga-deinterleave.pl sga-joinedpe
+        sga-mergeDriver.pl \
+        sga-asqg2dot.pl \
+        sga-beetl-index.pl \
+        sga-diffCalls.pl \
+        sga-popcat.pl \
+        sga-rename.pl \
+        sga-align \
+        sga-deinterleave.pl \
+        sga-joinedpe
+dist_noinst_SCRIPTS=
 
