--- bin/Makefile.am	2015-12-07 14:31:10.473366707 -0500
+++ bin/Makefile.am	2015-12-07 14:31:27.037537959 -0500
@@ -1,6 +1,17 @@
-dist_bin_SCRIPTS = \
-	sga-astat.py \
-	sga-bam2de.pl \
-    sga-mergeDriver.pl 
-dist_noinst_SCRIPTS=sga-align sga-deinterleave.pl sga-joinedpe
+dist_bin_SCRIPTS = \
+        sga-astat.py \
+        sga-bam2de.pl \
+        sga-mergeDriver.pl \
+        sga-asqg2dot.pl \
+        sga-beetl-index.pl \
+        sga-diffCalls.pl \
+        sga-popcat.pl \
+        sga-rename.pl \
+        sga-align \
+        sga-deinterleave.pl \
+        sga-joinedpe \
+        sga-call-variants.pl \
+        sga-preqc-report.py \
+        sga-vcf-dedup.pl
+dist_noinst_SCRIPTS=
 
