diff -ru sga-0.9.20-orig/src/bin/Makefile.am sga-0.9.20/src/bin/Makefile.am
--- sga-0.9.20-orig/src/bin/Makefile.am	2012-08-28 14:32:09.302789353 -0400
+++ sga-0.9.20/src/bin/Makefile.am	2012-08-28 14:33:29.152903168 -0400
@@ -1,6 +1,14 @@
 dist_bin_SCRIPTS = \
 	sga-astat.py \
 	sga-bam2de.pl \
-    sga-mergeDriver.pl 
-dist_noinst_SCRIPTS=sga-align sga-deinterleave.pl sga-joinedpe
+	sga-mergeDriver.pl \
+	sga-asqg2dot.pl \
+	sga-beetl-index.pl \
+	sga-diffCalls.pl \
+	sga-popcat.pl \
+	sga-rename.pl \
+	sga-align \
+	sga-deinterleave.pl \
+	sga-joinedpe
+dist_noinst_SCRIPTS=
 
