--- idba-1.1.1/src/sequence/short_sequence.h	2014-02-06 13:39:28.397761446 -0500
+++ idba-1.1.1/src/sequence/short_sequence2.h	2014-12-11 12:25:26.927809847 -0500
@@ -99,7 +99,7 @@
         return size() < seq.size();
     }
 
-    static const uint32_t kMaxShortSequence = 128;
+    static const uint32_t kMaxShortSequence = 300;
     static const uint32_t kNumBytes = (kMaxShortSequence + 3) / 4 + 2;
 
 private:
