--- vcftools_0.1.9-orig/Makefile	2012-05-10 01:52:12.000000000 -0400
+++ vcftools_0.1.9/Makefile	2014-07-11 16:37:25.622113648 -0400
@@ -20,9 +20,22 @@
 export MODDIR = ${PREFIX}/lib/perl5/site_perl
 
 DIRS = cpp perl
-install:
-	    @mkdir -p $(BINDIR); mkdir -p $(MODDIR); \
-        for dir in $(DIRS); do cd $$dir && $(MAKE) $(MAKEFLAGS) && cd ..; done
 
-clean:
-		@for dir in $(DIRS); do cd $$dir && $(MAKE) clean && cd ..; done
+INSTALLDIRS=$(DIRS:%=install-%)
+CLEANDIRS=$(DIRS:%=clean-%)
+
+.PHONY: subdirs $(INSTALLDIRS)
+.PHONY: subdirs $(CLEANDIRS)
+
+install: setup $(INSTALLDIRS)
+
+setup:
+	@mkdir -p $(BINDIR); 
+	@mkdir -p $(MODDIR);
+
+$(INSTALLDIRS):
+	$(MAKE) -C $(@:install-%=%)
+
+clean: $(CLEANDIRS)
+$(CLEANDIRS):
+	$(MAKE) -C $(@:clean-%=%) clean
