diff -ruN bowtie2-2.3.4.3-orig/scripts/convert_quals.pl bowtie2-2.3.4.3/scripts/convert_quals.pl
--- bowtie2-2.3.4.3-orig/scripts/convert_quals.pl	2018-10-12 13:45:46.168807278 -0400
+++ bowtie2-2.3.4.3/scripts/convert_quals.pl	2018-10-12 13:47:10.389768168 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 #
 # Copyright 2011, Ben Langmead <langmea@cs.jhu.edu>
diff -ruN bowtie2-2.3.4.3-orig/scripts/gen_2b_occ_lookup.pl bowtie2-2.3.4.3/scripts/gen_2b_occ_lookup.pl
--- bowtie2-2.3.4.3-orig/scripts/gen_2b_occ_lookup.pl	2018-10-12 13:45:46.183834709 -0400
+++ bowtie2-2.3.4.3/scripts/gen_2b_occ_lookup.pl	2018-10-12 13:47:10.418224985 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 #
 # Copyright 2011, Ben Langmead <langmea@cs.jhu.edu>
diff -ruN bowtie2-2.3.4.3-orig/scripts/gen_occ_lookup.pl bowtie2-2.3.4.3/scripts/gen_occ_lookup.pl
--- bowtie2-2.3.4.3-orig/scripts/gen_occ_lookup.pl	2018-10-12 13:45:46.583797150 -0400
+++ bowtie2-2.3.4.3/scripts/gen_occ_lookup.pl	2018-10-12 13:47:10.523583534 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 #
 # Copyright 2011, Ben Langmead <langmea@cs.jhu.edu>
diff -ruN bowtie2-2.3.4.3-orig/scripts/gen_solqual_lookup.pl bowtie2-2.3.4.3/scripts/gen_solqual_lookup.pl
--- bowtie2-2.3.4.3-orig/scripts/gen_solqual_lookup.pl	2018-10-12 13:45:46.673282993 -0400
+++ bowtie2-2.3.4.3/scripts/gen_solqual_lookup.pl	2018-10-12 13:47:10.551019306 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 #
 # Copyright 2011, Ben Langmead <langmea@cs.jhu.edu>
diff -ruN bowtie2-2.3.4.3-orig/scripts/infer_fraglen.pl bowtie2-2.3.4.3/scripts/infer_fraglen.pl
--- bowtie2-2.3.4.3-orig/scripts/infer_fraglen.pl	2018-10-12 13:45:46.907861794 -0400
+++ bowtie2-2.3.4.3/scripts/infer_fraglen.pl	2018-10-12 13:47:10.820458364 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 #
 # Copyright 2011, Ben Langmead <langmea@cs.jhu.edu>
diff -ruN bowtie2-2.3.4.3-orig/scripts/sim/AlignmentCheck.pm bowtie2-2.3.4.3/scripts/sim/AlignmentCheck.pm
--- bowtie2-2.3.4.3-orig/scripts/sim/AlignmentCheck.pm	2018-10-12 13:45:46.730605165 -0400
+++ bowtie2-2.3.4.3/scripts/sim/AlignmentCheck.pm	2018-10-12 13:47:10.579864061 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 #
 # Copyright 2011, Ben Langmead <langmea@cs.jhu.edu>
diff -ruN bowtie2-2.3.4.3-orig/scripts/sim/DNA.pm bowtie2-2.3.4.3/scripts/sim/DNA.pm
--- bowtie2-2.3.4.3-orig/scripts/sim/DNA.pm	2018-10-12 13:45:46.775613796 -0400
+++ bowtie2-2.3.4.3/scripts/sim/DNA.pm	2018-10-12 13:47:10.668077055 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 #
 # Copyright 2011, Ben Langmead <langmea@cs.jhu.edu>
diff -ruN bowtie2-2.3.4.3-orig/scripts/sim/Mutate.pm bowtie2-2.3.4.3/scripts/sim/Mutate.pm
--- bowtie2-2.3.4.3-orig/scripts/sim/Mutate.pm	2018-10-12 13:45:46.745824110 -0400
+++ bowtie2-2.3.4.3/scripts/sim/Mutate.pm	2018-10-12 13:47:10.609976805 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 #
 # Copyright 2011, Ben Langmead <langmea@cs.jhu.edu>
diff -ruN bowtie2-2.3.4.3-orig/scripts/sim/RandDNA.pm bowtie2-2.3.4.3/scripts/sim/RandDNA.pm
--- bowtie2-2.3.4.3-orig/scripts/sim/RandDNA.pm	2018-10-12 13:45:46.850868429 -0400
+++ bowtie2-2.3.4.3/scripts/sim/RandDNA.pm	2018-10-12 13:47:10.751534228 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 #
 # Copyright 2011, Ben Langmead <langmea@cs.jhu.edu>
diff -ruN bowtie2-2.3.4.3-orig/scripts/sim/run.pl bowtie2-2.3.4.3/scripts/sim/run.pl
--- bowtie2-2.3.4.3-orig/scripts/sim/run.pl	2018-10-12 13:45:46.790653775 -0400
+++ bowtie2-2.3.4.3/scripts/sim/run.pl	2018-10-12 13:47:10.695944145 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 #
 # Copyright 2011, Ben Langmead <langmea@cs.jhu.edu>
diff -ruN bowtie2-2.3.4.3-orig/scripts/sim/SampleRead.pm bowtie2-2.3.4.3/scripts/sim/SampleRead.pm
--- bowtie2-2.3.4.3-orig/scripts/sim/SampleRead.pm	2018-10-12 13:45:46.865829495 -0400
+++ bowtie2-2.3.4.3/scripts/sim/SampleRead.pm	2018-10-12 13:47:10.792073786 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 #
 # Copyright 2011, Ben Langmead <langmea@cs.jhu.edu>
diff -ruN bowtie2-2.3.4.3-orig/scripts/sim/Sim.pm bowtie2-2.3.4.3/scripts/sim/Sim.pm
--- bowtie2-2.3.4.3-orig/scripts/sim/Sim.pm	2018-10-12 13:45:46.805879054 -0400
+++ bowtie2-2.3.4.3/scripts/sim/Sim.pm	2018-10-12 13:47:10.724137938 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 #
 # Copyright 2011, Ben Langmead <langmea@cs.jhu.edu>
diff -ruN bowtie2-2.3.4.3-orig/scripts/sim/Test.pm bowtie2-2.3.4.3/scripts/sim/Test.pm
--- bowtie2-2.3.4.3-orig/scripts/sim/Test.pm	2018-10-12 13:45:46.760763837 -0400
+++ bowtie2-2.3.4.3/scripts/sim/Test.pm	2018-10-12 13:47:10.637474293 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 #
 # Copyright 2011, Ben Langmead <langmea@cs.jhu.edu>
diff -ruN bowtie2-2.3.4.3-orig/scripts/test/DNA.pm bowtie2-2.3.4.3/scripts/test/DNA.pm
--- bowtie2-2.3.4.3-orig/scripts/test/DNA.pm	2018-10-12 13:45:46.240645994 -0400
+++ bowtie2-2.3.4.3/scripts/test/DNA.pm	2018-10-12 13:47:10.447473364 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 #
 # Copyright 2011, Ben Langmead <langmea@cs.jhu.edu>
diff -ruN bowtie2-2.3.4.3-orig/scripts/test/simple_tests.pl bowtie2-2.3.4.3/scripts/test/simple_tests.pl
--- bowtie2-2.3.4.3-orig/scripts/test/simple_tests.pl	2018-10-12 13:45:46.567550450 -0400
+++ bowtie2-2.3.4.3/scripts/test/simple_tests.pl	2018-10-12 13:47:10.483257603 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl
 
 #
 # Copyright 2011, Ben Langmead <langmea@cs.jhu.edu>
