diff -urN allpathslg-52488-orig/src/allpaths_cache/CacheGroups.pl allpathslg-52488/src/allpaths_cache/CacheGroups.pl
--- allpathslg-52488-orig/src/allpaths_cache/CacheGroups.pl	2018-10-10 13:10:30.383179892 -0400
+++ allpathslg-52488/src/allpaths_cache/CacheGroups.pl	2018-10-10 13:10:48.062535392 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 ###############################################################################
 #                    SOFTWARE COPYRIGHT NOTICE AGREEMENT                      #
 #        This software and its documentation are copyright (2011) by the      #
diff -urN allpathslg-52488-orig/src/allpaths_cache/CacheLibs.pl allpathslg-52488/src/allpaths_cache/CacheLibs.pl
--- allpathslg-52488-orig/src/allpaths_cache/CacheLibs.pl	2018-10-10 13:10:30.491778085 -0400
+++ allpathslg-52488/src/allpaths_cache/CacheLibs.pl	2018-10-10 13:10:48.089112132 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 ###############################################################################
 #                    SOFTWARE COPYRIGHT NOTICE AGREEMENT                      #
 #        This software and its documentation are copyright (2011) by the      #
diff -urN allpathslg-52488-orig/src/allpaths_cache/CacheToAllPathsInputs.pl allpathslg-52488/src/allpaths_cache/CacheToAllPathsInputs.pl
--- allpathslg-52488-orig/src/allpaths_cache/CacheToAllPathsInputs.pl	2018-10-10 13:10:30.430560031 -0400
+++ allpathslg-52488/src/allpaths_cache/CacheToAllPathsInputs.pl	2018-10-10 13:10:48.114926288 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 ###############################################################################
 #                    SOFTWARE COPYRIGHT NOTICE AGREEMENT                      #
 #        This software and its documentation are copyright (2011) by the      #
diff -urN allpathslg-52488-orig/src/allpaths_cache/CacheToReads.pl allpathslg-52488/src/allpaths_cache/CacheToReads.pl
--- allpathslg-52488-orig/src/allpaths_cache/CacheToReads.pl	2018-10-10 13:10:30.446267582 -0400
+++ allpathslg-52488/src/allpaths_cache/CacheToReads.pl	2018-10-10 13:10:48.140754885 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 ###############################################################################
 #                    SOFTWARE COPYRIGHT NOTICE AGREEMENT                      #
 #        This software and its documentation are copyright (2011) by the      #
diff -urN allpathslg-52488-orig/src/allpaths_cache/ConvertToFastbQualb.pl allpathslg-52488/src/allpaths_cache/ConvertToFastbQualb.pl
--- allpathslg-52488-orig/src/allpaths_cache/ConvertToFastbQualb.pl	2018-10-10 13:10:30.400377632 -0400
+++ allpathslg-52488/src/allpaths_cache/ConvertToFastbQualb.pl	2018-10-10 13:10:48.173774595 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 ###############################################################################
 #                    SOFTWARE COPYRIGHT NOTICE AGREEMENT                      #
 #        This software and its documentation are copyright (2012) by the      #
diff -urN allpathslg-52488-orig/src/allpaths_cache/naifdb-test.pl allpathslg-52488/src/allpaths_cache/naifdb-test.pl
--- allpathslg-52488-orig/src/allpaths_cache/naifdb-test.pl	2018-10-10 13:10:30.368003609 -0400
+++ allpathslg-52488/src/allpaths_cache/naifdb-test.pl	2018-10-10 13:10:48.212575537 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w 
+#!/usr/bin/env perl -w 
 
 use strict;
 use NaifDB;
diff -urN allpathslg-52488-orig/src/allpaths_cache/OldPicardCacheToCache.pl allpathslg-52488/src/allpaths_cache/OldPicardCacheToCache.pl
--- allpathslg-52488-orig/src/allpaths_cache/OldPicardCacheToCache.pl	2018-10-10 13:10:30.461636480 -0400
+++ allpathslg-52488/src/allpaths_cache/OldPicardCacheToCache.pl	2018-10-10 13:10:48.239111099 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 ###########################################################################
 #       This software and its documentation are copyright (2009) by the   #
 #   Broad Institute/Massachusetts Institute of Technology.  All rights    #
diff -urN allpathslg-52488-orig/src/allpaths_cache/PicardToCache.pl allpathslg-52488/src/allpaths_cache/PicardToCache.pl
--- allpathslg-52488-orig/src/allpaths_cache/PicardToCache.pl	2018-10-10 13:10:30.506947236 -0400
+++ allpathslg-52488/src/allpaths_cache/PicardToCache.pl	2018-10-10 13:10:48.264817312 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 ###########################################################################
 #       This software and its documentation are copyright (2009) by the   #
 #   Broad Institute/Massachusetts Institute of Technology.  All rights    #
diff -urN allpathslg-52488-orig/src/allpaths_cache/PrepareAllPathsInputs.pl allpathslg-52488/src/allpaths_cache/PrepareAllPathsInputs.pl
--- allpathslg-52488-orig/src/allpaths_cache/PrepareAllPathsInputs.pl	2018-10-10 13:10:30.415389279 -0400
+++ allpathslg-52488/src/allpaths_cache/PrepareAllPathsInputs.pl	2018-10-10 13:10:48.291746997 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 ###############################################################################
 #                    SOFTWARE COPYRIGHT NOTICE AGREEMENT                      #
 #        This software and its documentation are copyright (2011) by the      #
diff -urN allpathslg-52488-orig/src/allpaths_cache/QualsStatsAddToCache.pl allpathslg-52488/src/allpaths_cache/QualsStatsAddToCache.pl
--- allpathslg-52488-orig/src/allpaths_cache/QualsStatsAddToCache.pl	2018-10-10 13:10:30.537261742 -0400
+++ allpathslg-52488/src/allpaths_cache/QualsStatsAddToCache.pl	2018-10-10 13:10:48.317124606 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 ###############################################################################
 #                    SOFTWARE COPYRIGHT NOTICE AGREEMENT                      #
 #        This software and its documentation are copyright (2011) by the      #
diff -urN allpathslg-52488-orig/src/kmers/KmerSpectrumPlot.pl allpathslg-52488/src/kmers/KmerSpectrumPlot.pl
--- allpathslg-52488-orig/src/kmers/KmerSpectrumPlot.pl	2018-10-10 13:10:29.131593155 -0400
+++ allpathslg-52488/src/kmers/KmerSpectrumPlot.pl	2018-10-10 13:10:48.343612890 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 ###############################################################################
 #                    SOFTWARE COPYRIGHT NOTICE AGREEMENT                      #
 #        This software and its documentation are copyright (2011) by the      #
diff -urN allpathslg-52488-orig/src/paths/ErrorCorrectReads.pl allpathslg-52488/src/paths/ErrorCorrectReads.pl
--- allpathslg-52488-orig/src/paths/ErrorCorrectReads.pl	2018-10-10 13:10:24.814440401 -0400
+++ allpathslg-52488/src/paths/ErrorCorrectReads.pl	2018-10-10 13:10:48.369144993 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 ###############################################################################
 #                    SOFTWARE COPYRIGHT NOTICE AGREEMENT                      #
 #        This software and its documentation are copyright (2012) by the      #
diff -urN allpathslg-52488-orig/src/paths/HaploidifyStatsPlot.pl allpathslg-52488/src/paths/HaploidifyStatsPlot.pl
--- allpathslg-52488-orig/src/paths/HaploidifyStatsPlot.pl	2018-10-10 13:10:20.296165067 -0400
+++ allpathslg-52488/src/paths/HaploidifyStatsPlot.pl	2018-10-10 13:10:48.395250796 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 ###############################################################################
 #                    SOFTWARE COPYRIGHT NOTICE AGREEMENT                      #
 #        This software and its documentation are copyright (2011) by the      #
diff -urN allpathslg-52488-orig/src/reporting/CompileAssemblyReport.pl allpathslg-52488/src/reporting/CompileAssemblyReport.pl
--- allpathslg-52488-orig/src/reporting/CompileAssemblyReport.pl	2018-10-10 13:10:18.589204744 -0400
+++ allpathslg-52488/src/reporting/CompileAssemblyReport.pl	2018-10-10 13:10:48.421538581 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 #///////////////////////////////////////////////////////////////////////////////
 #//                   SOFTWARE COPYRIGHT NOTICE AGREEMENT                     //
 #//       This software and its documentation are copyright (2009) by the     //
diff -urN allpathslg-52488-orig/src/reporting/ParseMemMonitorOutput.pl allpathslg-52488/src/reporting/ParseMemMonitorOutput.pl
--- allpathslg-52488-orig/src/reporting/ParseMemMonitorOutput.pl	2018-10-10 13:10:18.542794707 -0400
+++ allpathslg-52488/src/reporting/ParseMemMonitorOutput.pl	2018-10-10 13:10:48.452002525 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 #///////////////////////////////////////////////////////////////////////////////
 #//                   SOFTWARE COPYRIGHT NOTICE AGREEMENT                     //
 #//       This software and its documentation are copyright (2009) by the     //
diff -urN allpathslg-52488-orig/src/util/HelpMe.pl allpathslg-52488/src/util/HelpMe.pl
--- allpathslg-52488-orig/src/util/HelpMe.pl	2018-10-10 13:10:15.148941920 -0400
+++ allpathslg-52488/src/util/HelpMe.pl	2018-10-10 13:10:48.480349993 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 ###############################################################################
 #                    SOFTWARE COPYRIGHT NOTICE AGREEMENT                      #
 #        This software and its documentation are copyright (2012) by the      #
diff -urN allpathslg-52488-orig/src/util/QualbStatsPlot.pl allpathslg-52488/src/util/QualbStatsPlot.pl
--- allpathslg-52488-orig/src/util/QualbStatsPlot.pl	2018-10-10 13:10:15.582339370 -0400
+++ allpathslg-52488/src/util/QualbStatsPlot.pl	2018-10-10 13:10:48.506391048 -0400
@@ -1,4 +1,4 @@
-#!/usr/bin/perl -w
+#!/usr/bin/env perl -w
 ###############################################################################
 #                    SOFTWARE COPYRIGHT NOTICE AGREEMENT                      #
 #        This software and its documentation are copyright (2011) by the      #
